### PR TITLE
Bumps Stripe from 10.10.0 to 10.11.0

### PR DESCRIPTION
**Pull request by filipefurtad0**
#### What? Why?

Bumps Stripe from 10.10.0 to 10.11.0

#### What should we test?

- green build

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [ ] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [x] Technical changes only
- [ ] Feature toggled

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: .env.test [hunk 1]
```diff
@@ -14,3 +14,6 @@ SITE_URL="test.host"
 
 OPENID_APP_ID="test-provider"
 OPENID_APP_SECRET="12345"
+
+CLIENT_SECRET =~ /secret.+/
+HOSTNAME =~ /"hostname":".+"/
```
Comment (filipefurtad0) [inline .env.test:19]: I agree, both client_secret and data on hostname should not be public.

I've tried this approach to hide sensitive data, but it does not seem to work. I'm not sure the expressions are being evaluated correctly.

Any suggestions @dacook ?

Comment (mkllnk) [reply .env.test]: We can't use patterns here. And I couldn't find any use of patterns in the docs. We have to provide the real values to VCR while recording. We do that by fetching the secrets from the environment and tell VCR to filter those values.

So we would need to find out how the hostname is computed and do the same. And where does the client secret come from?

Comment (rioug) [reply .env.test]: Fixed it ! see https://github.com/openfoodfoundation/openfoodnetwork/pull/12252/commits/a33eb80f56c4fc80c0d38086982019c6b8bb4b69

**Review by mkllnk — APPROVED** [review]
Great!

And thank you for your initiative. Excellent.

File: spec/support/vcr_setup.rb [hunk 2]
```diff
@@ -16,9 +16,13 @@
...
-    HOSTNAME
   ].each do |env_var|
     config.filter_sensitive_data("<HIDDEN-#{env_var}>") { ENV.fetch(env_var, nil) }
   end
+  config.filter_sensitive_data('<HIDDEN-STRIPE-USER-AGENT>') { |interaction|
+    interaction.request.headers['X-Stripe-Client-User-Agent']&.public_send(:[], 0)
+  }
+  config.filter_sensitive_data('<HIDDEN-CLIENT-SECRET>') { |interaction|
+    interaction.response.body.match(/"client_secret": "(pi_.+)"/)&.public_send(:[], 1)
+  }
```
Comment (dacook) [inline spec/support/vcr_setup.rb:27]: Clever idea!

**Pull request merged**